generators/typescript/sdk/endpoint-error-union-generator/src/GeneratedEndpointErrorUnionImpl.ts

```diff
@@ -73,7 +76,9 @@ export class GeneratedEndpointErrorUnionImpl implements GeneratedEndpointErrorUn
             includeOtherInUnionTypes: true,
             includeSerdeLayer,
             noOptionalProperties,
-            retainOriginalCasing
+            retainOriginalCasing,
+            inlineInlineTypes,
+            inline: false
```

Comment:
Is this intentional? If so, a lightweight comment justifying why we always use `false` here might be useful.